spec/features/user_logs_in_spec.rb

```diff
@@ -53,5 +53,26 @@
       click_link_or_button "Log In"
       expect(page).to have_content("#{borrower.name}'s Projects")
     end
+
+    scenario "cannot see log in button" do
+      visit '/'
+      fill_in('session[email]', with: borrower.email)
+      fill_in('session[password]', with: borrower.password)
+      click_link_or_button 'Log In'
+
+      expect(page).to_not have_content('Log In')
+      expect(page).to have_content('Log out')
+    end
+
+    scenario "can log out" do
+      visit '/'
+      fill_in('session[email]', with: borrower.email)
+      fill_in('session[password]', with: borrower.password)
+      click_link_or_button 'Log In'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.